test/end-to-end/offer.js

```diff
@@ -1396,6 +1396,113 @@ describe("Offer endpoint tests", () => {
         });
     });
 
+    describe("GET /offers/:companyId", () => {
```

Comment:
I think this line should be `describe("GET /offers/company/:companyId", () => {`, but it's not important.

Reply:
It matters! It should match the endpoint. you'll thank it later when debugging tests